pkg/pgmodel/ingestor/ingestor.go

```diff
@@ -43,33 +31,42 @@ type DBIngestor struct {
 	closed     *atomic.Bool
 }
 
+type Config struct {
+	NumCopiers              int
+	InvertedLabelsCacheSize uint64
+	Flags                   *Flags
```

Comment:
Not sure how much I like this. I'd rather keep all Ingestor configuration in one struct. Check my other comment.